dds/idl/be_produce.cpp

```diff
@@ -107,6 +108,22 @@ BE_abort()
 
 namespace {
 
+///Only warn once per run
+bool pragmaWarn = false;
+/**
+ * @brief prints a warning if a #pragma is detected based off of
+ * IDL_GlobalData::dcps_type_info_map_::cur_size_
+ * @author ceneblock
+ */
+void warnOnPragma()
+{
+  std::cerr << "WARNING: #pragma is discontinued\n" <<
+               "Use annotations for future uses\n" <<
+               "See: https://www.omg.org/spec/DDS-XTypes/About-DDS-XTypes/\n";
```

Comment:
At some point we'll have our own documentation on the annotations and it would be better to point to that. You don't have to change this now since we don't have that yet.

Reply:
Should I change the website to a`#define ANNOTATION_WEBSITE` so we don't have to go searching for this in the future and it's a one and done?

Reply:
It will be in the Developer's Guide, so it won't be a website, but regardless I don't think it will be necessary.